libc/spec/posix.td

```diff
@@ -567,16 +572,6 @@ def POSIX : StandardSpec<"POSIX"> {
           RetValSpec<IntType>,
           [ArgSpec<ConstCharPtr>]
         >,
-        FunctionSpec<
```

Comment:
why were these deleted?

Reply:
They were repeated somehow with different types. There are already definitions using `PidT` in the above.